src/Components/FlashSalesCountdownSection/FlashSalesCountdownSection.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Countdown from "./Countdown/Countdown";
+import ArrowButtons from "./ArrowButtons/ArrowButtons";
+import styles from "./FlashSalesCountdownSection.module.css";
+import Title from "./Title/Title";
+import Heading from "./Heading/Heading";
+const SecondSection = () => {
```

Comment:
The name of the constant is not identical to the file's name

Reply:
done.